doc/_scripts/redirects.py

```diff
@@ -13,136 +13,6 @@
 """
 
 REDIRECTS = [
-    ('api/api', 'reference/index'),
```

Comment:
I'm not sure about this one; seems nice to have these old redirects in place. Maybe separate them into an OLD_REDIRECTS variable that gets appended onto REDIRECTS for clarity for now instead?

Reply:
I'm not sure if it's worth to keep a list of redirects for docs we do not list (deleted ones apply to ≤ 1.13.0):

![image](https://user-images.githubusercontent.com/25011557/125753173-186f7d64-542d-4dc5-ad53-f6943f47f519.png)

Reply:
I understand they're not listed but people still have old links in their bookmarks.

Reply:
I think we should have a certain policy here, we should definitely avoid keeping old redirects forever. @nashif ?

Reply:
We can always stick them in another file that we load from conf.py if you're worried about the size of the list. It seems like a cheap and easy thing to do with minimal maintenance burden, no?

Reply:
Dropped the commit for now, I prefer to not block other changes while this is decided.